metrics/web/docs/02_presentation_layer_architecture.md

```diff
@@ -53,11 +129,23 @@ No blockers.
 
 > What will be impacted by the project?
 
+- The implementation of presentation layer components of the Metrics Web Application is impacted.
+- Since a developer should consider creating low-level widgets and then use it for creating the desired module-specific one, hence the process of delivering a presentation layer component is impacted. 
+
 # Testing
 > How will the project be tested?
 
+The presentation layer will be unit-tested and integration-tested using the core [flutter_test](https://api.flutter.dev/flutter/flutter_test/flutter_test-library.html) package and the [flutter_driver](https://api.flutter.dev/flutter/flutter_driver/flutter_driver-library.html) package respectively.
+
 # Alternatives Considered
 > Summarize alternative designs (pros & cons)
 
+- Not document the presentation layer of the Metrics Web Application:
+    - Cons: 
+        - As the application grows, the future development of new modules and maintaining the old ones may be tricky without any document describing the presentation layer.
+        - As there is no explanation in differences between top-level and low-level widgets in the context of the Metrics Web Application, newcomers will be required to investigate this by themselves. This point makes the Metrics Web Application entry threshold quite hight.
```

Comment:
```suggestion
        - With no written explanation in differences between top-level and low-level widgets in the context of the Metrics Web Application, it will be longer for the new team members to become productive.
```